packages/toolbox-core/tests/test_client.py

```diff
@@ -92,7 +92,11 @@ async def test_load_tool_success(aioresponses, test_tool_str):
         assert callable(loaded_tool)
         # Assert introspection attributes are set correctly
         assert loaded_tool.__name__ == TOOL_NAME
-        assert loaded_tool.__doc__ == test_tool_str.description
+        assert (
+            loaded_tool.__doc__
+            == test_tool_str.description
+            + f"\n\nArgs:\n    param1 (string): Description of Param1"
```

Comment:
Optional: In case the above comment makes sense.
```suggestion
            + f"\n\nArgs:\n    param1 (str): Description of Param1"
```